docs/_legacy/angular.md

```diff
@@ -92,21 +85,3 @@ You can also set up a IntelliJ run configuration as outlined above. Instead of c
 ## Generate war files
 Execute `mvn package`.
 
-
-## Development Hints
-- [Exiting a for-each loop early](https://stackoverflow.com/a/2641374/6592788) 
-
-
-## License
-
-Copyright (c) 2017 Contributors to the Eclipse Foundation
```

Comment:
As stated by the Eclipse Handbook, you should include a copyright statement in all files if possible. Please add the copyright and license statement as a comment at the top.

This applies for all files.